crates/astria-sequencer/src/app/mod.rs

```diff
@@ -1175,7 +1175,17 @@ impl App {
                     .iter()
                     .any(|act| act.is_fee_asset_change() || act.is_fee_change());
 
-        Ok(state_tx.apply().1)
+        // index all event attributes
+        let mut events = state_tx.apply().1;
+        for event in &mut events {
+            event
+                .attributes
+                .iter_mut()
+                .filter(|attr| !attr.index)
```

Comment:
Not sure there's any point to this `filter`?

Reply:
Yeah, that filter is useless.